#3 match documentation (#14)
abb-motors-and-generators/smart-sensor-api-client: use_case_1/README.md

# Use Case 1: Request Asset Details

> With this use case, the different kind of asset data can be requested. Find below three example scripts that can be executed.

Reviewer: ```suggestion
With this use case, you can request different kinds of asset data (examples below).
```
